playwright-e2e/fixtures/singular-fixture.ts

```diff
@@ -15,6 +17,14 @@ type Fixtures = {
  */
 // This fixture runs per test when called. Removes duplicated code in every test.
 const fixture = baseTest.extend<Fixtures>({
+  page: async ({ baseURL, page }, use) => {
+    await page.route('**/*', (route) => {
+      return REQUEST_EXCLUDES.some((urlPart) => route.request().url().includes(urlPart))
+        ? route.abort()
+        : route.continue();
+    });
+    await use(page);
+  },
```

Comment:
Could you summarize what's going on here? I see it's performing filtering based on some URL pattern, but I'm unsure what the intent of the filtering is.

Reply:
General good test automation practice, abort seding non-product requests. A little bit, it improves page performance, decrease network bandwidth usage.

For now, intercept and abort all Google `https://www.google-analytics.com/` requests because analytic data generated by automation tests are generally not meaningful and useful.

Reply:
Thank you!